ethereum/core/src/main/java/org/hyperledger/besu/ethereum/privacy/storage/PrivateStateStorage.java

```diff
@@ -44,13 +37,8 @@
 
   interface Updater {
 
-    Updater putTransactionLogs(Bytes32 transactionHash, List<Log> logs);
-
-    Updater putTransactionResult(Bytes32 transactionHash, Bytes events);
-
-    Updater putTransactionStatus(Bytes32 transactionHash, Bytes status);
-
-    Updater putTransactionRevertReason(Bytes32 txHash, Bytes bytesValue);
+    Updater putTransactionReceipt(
+        Bytes blockHashTransactionHash, PrivateTransactionReceipt receipt);
```

Comment:
As I mentioned before, I think the logic of concatenating the block hasn and the transaction hash should be implemented in this method. And this should look like:
```
Updater putTransactionReceipt(
        Bytes blockHash, transactionHash, PrivateTransactionReceipt receipt);
```